Example/APMAlertController/ViewController.swift

```diff
@@ -140,18 +137,15 @@ class ViewController: UIViewController {
         let message = "Message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message message." /*"This is message. One, Two. Message."*/
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/almas-dev/APMAlertController/blob/458898c990446c048fd2759812785a8e78fcda83/Example/APMAlertController/ViewController.swift#L137"></span>Line should be 150 characters or less: currently 634 characters</td>
    </tr>
  </tbody>
</table>